timon/plugins/base.py

```diff
@@ -47,3 +49,22 @@ async def stop_plugin(self):
     async def stop(self):
         raise NotImplementedError(
             "Plugin %s doesn't have a start method implemented", self.name)
+
+
+class OnDbStorePlugin:
+    """ if a plugin also derives from this class,
+        it will be invoked when a prob's state was updated and is being stored
```

Comment:
if the class is called `OnDbStorePlugin` then the comment could be `is being stored in db`
if DB is secondary then even the class might have the `Db` removed from its name

Reply:
class and method names are obvious enough